lib/HALSim/src/Gps.cpp

```diff
@@ -94,6 +97,36 @@ bool Gps::getPosition(int32_t& xPos, int32_t& yPos)
     return isPositionValid;
 }
 
+bool Gps::getOrientation(int32_t& orientation)
+{
+    bool isOrientationValid = false;
+
+    if (nullptr != m_compass)
+    {
+        /* Get orientation in radians from Webots. */
+        const double* orientationVector = m_compass->getValues();
+
+        if (nullptr != orientationVector)
+        {
+            double xVector = orientationVector[0];
+            double yVector = orientationVector[1];
```

Comment:
The names `xVector` and `yVector` are confusing --> sound multi-dimensional

Reply:
It is multi-dimensional. X and Y directions :D

Reply:
`xVector` and `yVector` are one-dimensional variables, but the `vector` inside the name make them sound multi-dimensional --> maybe something with `component` or `direction`?